src/adapters/postgres/cohort-adapter.ts

```
@@ -904,63 +909,90 @@ export class PostgresCohortService {
+                case 'calendar':
+                  params.push(`%${value}%`);
+                  conditions.push(
+                    `(f."name" = $${paramIndex} AND (fv."calendarValue"::text LIKE $${
+                      paramIndex + 1
+                    } OR fv."value" LIKE $${paramIndex + 1}))`
+                  );
+                  paramIndex += 2;
+                  break;
+                case 'checkbox':
+                  const boolValue =
+                    value === 'true' || value === '1' || value === true;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)